### ISSUE TITLE
non Latin-1 characters in stopwords

### ISSUE THREAD

**Issue by msenger** [xenoterracide/Dist-Zilla-Plugin-Test-PodSpelling#6]
I wonder how to put into my dist.ini file a stopword "d'être"?. In my POD file, I am expressing it this way: raison d'E&lt;234&gt;tre.

Here is a dump what I have in my dist.ini (actually, a copy in a.tmp) :

<pre>
computer$ od -cxw80 a.tmp
0000000   s   t   o   p   w   o   r   d   s       =       d   ' 303 252   t   r   e  \n
           7473    706f    6f77    6472    2073    203d    2764    aac3    7274    0a65
</pre>

Thanks for any advise and help,
Martin (martin.senger@gmail.com)

**Comment by xenoterracide** [comment]
there are no limitations that I've imposed upon this, however I heard a rumor once that dzil is unicode handicapped, and to be honest after looking through Pod::Spell's code it seems to have the idea in mind that it will only suppport ascii. Unfortunately I don't know the answer to your question, or to be fair even if it's possible.

you might try [adding stopwords in pod](http://search.cpan.org/~sburke/Pod-Spell-1.01/Spell.pm#ADDING_STOPWORDS)

but if that doesn't work I'm guessing it's a bug on the level of Pod::Spell and I've tried to convince the author (maintainer?) to give me comaint https://rt.cpan.org/Public/Bug/Display.html?id=75609 so we can get it cleaned up, to no avail. (maybe if more people comment)
